packages/vite/package.json

```diff
@@ -123,7 +122,8 @@
   "peerDependencies": {
     "less": "*",
     "sass": "*",
-    "stylus": "*"
+    "stylus": "*",
+    "terser": "^5.13.1"
```

Comment:
I assume this would prevent terser 6 and up, should we use `*` also here?

Reply:
I think it is safer to prevent terser 6+ as it could include breaking changes.

Also vite assumes 5.13.1 and up (accurately it may be 5.0.0 or something). For example, this line.
https://github.com/vitejs/vite/blob/bb603d3d66506deb353e4d6346d5e8361fdb9e62/packages/vite/src/node/build.ts#L255-L256

So I think `>=5.13.1` is better than `*` if we would like not to prevent terser 6+.

Reply:
I think it is ok to leave it with `^5.x.x`. Once terser releases v6 we can update the peerDep after checking that there aren't breaking changes.

Reply:
We may want to find the lowest possible version, so we don't force users to e.g. get in conflict with two versions of terser because something else also requests terser.
But beside that, okay 👌